lib_xua/src/midi/queue.h

```diff
@@ -3,8 +3,14 @@
 #ifndef QUEUE_H_
 #define QUEUE_H_
 
+#include "midioutparse.h"
+
 #define assert(x) asm("ecallf %0"::"r"(x));
```

Comment:
Can we replace this with xassert (dev with get line number then). lib_xassert is already a dependency.

Reply:
done